modules/ROOT/pages/getting-started.adoc

```diff
@@ -493,6 +493,8 @@ Configuration properties:
 <3> The xref:shared-bucket-access.adoc[shared bucket access] feature allows Couchbase Server SDKs to also perform operations on this bucket.
 <4> `num_index_replicas` is the number of index replicas stored in Couchbase Server, introduced with xref:indexing.adoc[GSI/N1QL indexing].
 If you're running a single Couchbase Server node for development purposes the `num_index_replicas` must be set to `0`.
+<5> If the document body contains the `sdk` property, route it to a channel called `doc.sdk`, this feature is introduced in xref:sync-gateway-channels.adoc[Access Control].
+The sync function can be surrounded with backticks (+`+) or with standard JSON double quotes, but in that case new lines must be escaped (`\n`).
```

Comment:
```suggestion
The Sync Function can be surrounded with backticks (+`+) to allow multiple lines to be used; or with standard JSON double quotes, in which case new lines must be escaped (`\n`).
```

My thinking is that this makes it more explicit, but it could just be personal preference...